src/frontends/tests/frontend/shared/src/op_fuzzy.cpp

```diff
@@ -45,7 +45,7 @@ inline void addInputOutput(cnpy::NpyArray& npy_array, test::TestCase& test_case,
     if (is_input)
         test_case.add_input(data);
```

Comment:
`test_case.add_input(data);` changed to
`test_case.add_input(npy_array.shape, data);` the following line 68 ~ 80 should be removed.

Reply:
great!